builder/src/permissioned.rs

```diff
@@ -379,6 +382,8 @@ impl<N: network::Type, P: SequencerPersistence, Ver: StaticVersionType + 'static
         hotshot_builder_api_url: Url,
         max_api_timeout_duration: Duration,
         buffered_view_num_count: usize,
+        maximise_txns_count_timeout_duration: Duration,
```

Comment:
There are many arguments, I believe this is due for a refactor. Not necessarily something to solve in this PR.